scripts/setup_build/setup_build.vcxproj

```diff
@@ -270,13 +248,13 @@ popd
       <GenerateDebugInformation>true</GenerateDebugInformation>
     </Link>
     <PreBuildEvent>
-      <Command>powershell -NonInteractive -ExecutionPolicy Unrestricted $(SolutionDir)scripts\copy_pre_commit.ps1 $(SolutionDir)scripts\pre-commit
+      <Command Condition="'$(Platform)'=='x64'">powershell -NonInteractive -ExecutionPolicy Unrestricted $(SolutionDir)scripts\copy_pre_commit.ps1 $(SolutionDir)scripts\pre-commit
```

Comment:
Is the pre commit script x64-specific (for this and the below copy_pre_commit.ps1 calls)?

Reply:
Yes. See a few lines below in this command. It's copying the VC redist bits to the output folder, so it will need to select a platform specific set of bits.